src/transformers/models/rag/tokenization_rag.py

```diff
@@ -63,42 +62,18 @@ def __call__(self, *args, **kwargs):
     def batch_decode(self, *args, **kwargs):
         return self.generator.batch_decode(*args, **kwargs)
 
-    @add_start_docstrings(PREPARE_SEQ2SEQ_BATCH_DOCSTRING)
     def prepare_seq2seq_batch(
         self,
         src_texts: List[str],
         tgt_texts: Optional[List[str]] = None,
         max_length: Optional[int] = None,
         max_target_length: Optional[int] = None,
-        padding: str = "longest",
-        return_tensors: str = None,
-        truncation=True,
         **kwargs,
     ) -> BatchEncoding:
         if max_length is None:
             max_length = self.question_encoder.model_max_length
-        model_inputs: BatchEncoding = self.question_encoder(
-            src_texts,
-            add_special_tokens=True,
-            return_tensors=return_tensors,
-            max_length=max_length,
-            padding=padding,
-            truncation=truncation,
-            **kwargs,
-        )
-        if tgt_texts is None:
-            return model_inputs
-        # Process tgt_texts
         if max_target_length is None:
             max_target_length = self.generator.model_max_length
-        labels = self.generator(
-            tgt_texts,
-            add_special_tokens=True,
-            return_tensors=return_tensors,
-            padding=padding,
-            max_length=max_target_length,
-            truncation=truncation,
-            **kwargs,
-        )["input_ids"]
-        model_inputs["labels"] = labels
-        return model_inputs
+        return super().prepare_seq2seq_batch(
```

Comment:
Rag is a very special tokenizer, since it uses two different tokenizers for the `input_ids` and `labels`, but I think this can also be handled well by the context manager no @sgugger